editor/editor_node.cpp

```diff
@@ -6255,10 +6257,14 @@ bool EditorPluginList::forward_gui_input(const Transform2D &p_canvas_xform, cons
 	return discard;
 }
 
-bool EditorPluginList::forward_spatial_gui_input(Camera *p_camera, const Ref<InputEvent> &p_event) {
+bool EditorPluginList::forward_spatial_gui_input(Camera *p_camera, const Ref<InputEvent> &p_event, bool serve_when_force_input_enabled) {
 	bool discard = false;
 
 	for (int i = 0; i < plugins_list.size(); i++) {
+		if((!serve_when_force_input_enabled) && plugins_list[i]->is_input_event_forwarding_always_enabled()){
```

Comment:
That doesn't follow the style guide, see the Travis logs. You could install the pre-commit hook (see `misc/hooks/README.md`) so that clang-format is applied automatically on your commits.